README.md

```diff
@@ -112,14 +81,17 @@ The following options are specific to appmetrics:
 
 ## Running Node Application Metrics
 
-### The `node-hc` command
-If you [globally installed](#installation) this module with npm, you can use the `node-hc` command to run your application instead of the `node` command. This will run your application as it would normally under node (including any node options) but additionally load and start `appmetrics`.
+### The `node-hc` command (removed)
+In previous versions appmetrics came with an executable, `node-hc`, which could be used instead of the `node` command to run your application and load and start appmetrics. This has been removed in version 4.0.0, instead please use:
 
 ```sh
-$ node-hc app.js
+$ node --require appmetrics/start app.js
 ```
+to preload and start appmetrics, or use the NODE_OPTIONS environment variable:
 
-The purpose of this mode of operation is to provide monitoring of the application without requiring any changes to the application code. The data is sent to the Health Center Eclipse IDE client.
+```sh
+$ export NODE_OPTIONS="require appmetrics/start"
```

Comment:
I think this needs to be ` $ export NODE_OPTIONS="--require appmetrics/start"`, the idea being that you pass the same string to `NODE_OPTIONS` that you'd pass as a command line flag.

Reply:
thanks